File: Python/7py 3/spam.py
"""
|--------------------------------------------------------------------------
| Multiply characters
|--------------------------------------------------------------------------
|
"""

def spam(number):
   res = ''
   for i in range(number):
      res+='hue'
   return res


spam = 'hue'.__mul__


def spam(number):
   return 'hue' * number


def spam(number):
   return ''.join(['hue' for i in range(number)])


def spam(number):
   return ('{0}' * number).format('hue')


print(spam(1), "hue")
print(spam(6), "huehuehuehuehuehue")
print(spam(14), "huehuehuehuehuehuehuehuehuehuehuehuehuehue")

File: JavaScript/7js 3/spam.js
/*
|--------------------------------------------------------------------------
| Multiply characters
|--------------------------------------------------------------------------
|
*/

function spam(number) {
   let res = [];
   for (let i = 0; i < number; i++) {
      res.push('hue');
   }
   return res.join('');
}


function spam(number) {
   return Array(++number).join("hue");
}


function spam(number) {
   return "hue".repeat(number);
}


function spam(number) {
   var res = '';
   while (number--) {
      res += 'hue'
   }
   return res;
}


function spam(number) {
   return (number != 0) ? "hue" + spam(number - 1) : "";
}


console.log(spam(1), "hue");
console.log(spam(6), "huehuehuehuehuehue");
console.log(spam(14), "huehuehuehuehuehuehuehuehuehuehuehuehuehue");